life/modernized/01-preface.adoc

```diff
@@ -82,6 +82,7 @@ to whose perusal in much love and sincerity I recommend it, and all to the grace
 which is able to build them up,
 and give them an inheritance among them that are sanctified;
 which is the sincere desire of, reader,
+//I'm not sure where this letter begins.
```

Comment:
This isn't a letter. It's the preface of the journal, and the guy who  wrote the preface signed it. So we can leave this one alone.